src/api/controllers/likeDislikeController.js

```diff
@@ -0,0 +1,127 @@
+import models from '../models';
+import statusCode from '../../helpers/constants/status.codes';
+
+/**
+ * containing all like dislike comment 's  controllers
+ * @export
+ * @class LikeDislikeController
+ */
+export default class LikeDislikeCommentController {
+  /**
+   * add a new like or dislike
+   * @static
+   * @param {Object} req the request
+   * @param {Object} res the response to be sent
+   * @memberof LikeDislikeCommentController
+   * @returns {Object} res
+   */
+  static async likeComment(req, res) {
+    const { commentId } = req.params;
+    const { id } = req.user.user;
+    const hasDisliked = await models.LikeDislike.findOne({
+      where: {
+        commentId,
+        userId: id,
+        dislikes: 1,
+        likes: 0
+      }
+    });
+    if (hasDisliked) {
+      await models.LikeDislike.update({ dislikes: 0, likes: 1 },
+        { where: { id: hasDisliked.dataValues.id } });
+      return res.status(statusCode.OK).json({
+        status: statusCode.OK,
+        message: 'Thank you! You have successfully  liked this comment '
```

Comment:
let's find something appropriate here... @murediane why not just ``successful`` or something